tests/driver_isl29125/Makefile

```diff
@@ -1,4 +1,8 @@
 APPLICATION = driver_isl29125
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= samr21-xpro
```

Comment:
Why the preference for the `samr21-xpro` board? I have no strong opinion about this, but this line does not add any value to the PR itself (: